radicle-cli/src/terminal/textbox.rs

```diff
@@ -39,8 +39,10 @@ impl fmt::Display for TextBox {
             .max()
             .unwrap_or(0)
             + 2;
-        if term::width() < width + 2 {
-            width = term::width() - 2
+        if let Some(max) = term::width() {
+            if max < width + 2 {
+                width = max - 2
+            }
```

Comment:
It appears width is the subject therefor should it be

```
if let Some(max) = term::width() {
  if max < width + 2 {
    width = max - 2
  }
}
```

Reply:
Yes, it tripped me up actually, but didn't think it worth changing. I'll make the change.